components/navbar/style.css

```diff
@@ -0,0 +1,81 @@
+header {
+    background-color: #036;
+    border-bottom: 2px solid #fcba19;
+    padding: 0 30px 0 30px;
+    color: #fff;
+    display: flex;
+    height: 65px;
+    top: 0px;
+    position: fixed;
+    width: 100%;
```

Comment:
I think we probably won't need to set the width explicitly if the `.other` container has the `flex-grow: 1` property

Reply:
Fixed.